docs/modules/ROOT/pages/how-to-guides/proc_managed_services_onboarding.adoc

```diff
@@ -111,13 +82,11 @@ releaseStrategy: <release-strategy> <.>
 * Run the following command to copy the application from the development workspace to the managed workspace:
 
 +
-[source,]
+[source,shell]
 ----
 ./copy-application.sh <managed workspace name> -a <development workspace name>/<application name>
```

Comment:
Where do you get the script? Is this documented somewhere? I don't see any instructions in the prerequisites here. And on the previous page, it says "You have completed the steps listed in the [Getting started in the CLI](https://redhat-appstudio.github.io/docs.appstudio.io/Documentation/main/getting-started/getting_started_in_cli/) page." So I checked that page, but that page only describes installing `oc` and logging in.